utility_functions.h

```diff
@@ -27,6 +30,63 @@ double convolution(double alpha,
                    unsigned int phi_width,
                    const SAMRAI::pdat::CellIndex<NDIM>& idx,
                    const double* const dx);
+
+enum Kernel
```

Comment:
And this would be accessed as `IBAMR::Kernel`?

Reply:
Yes. Make sure you include `utility_functions.h`